Remix tutorial (trivial, but I think we should have something) (#71)
ethereum-optimism/optimism-tutorial: remix/README.md

# Using Optimistic Ethereum with the Remix Development Environment

[![Discord](https://img.shields.io/discord/667044843901681675.svg?color=768AD4&label=discord&logo=https%3A%2F%2Fdiscordapp.com%2Fassets%2F8c9701b98ad4372b58f13fd9f65f966e.svg)](https://discord.com/channels/667044843901681675)
[![Twitter Follow](https://img.shields.io/twitter/follow/optimismPBC.svg?label=optimismPBC&style=social)](https://twitter.com/optimismPBC)

This tutorial aims to help you get started with developing decentralized applications on [Optimistic Ethereum](https://optimism.io/) using [Remix](https://remix.ethereum.org/#optimize=false&runs=200&evmVersion=null). Applications 
running on top of Optimistic Ethereum are about as secure as those running on the underlying Ethereum mainnet itself, but are
[significantly cheaper](https://optimism.io/gas-comparison).

> :warning: We are currently in the process of upgrading to OVM 2.0, but that
> is still work in progress (expected to end 11 NOV 2021). This tutorial is
> already upgraded for OVM 2.0, but parts of it will change during the upgrade
> process.

   <!-- TEMP-OVM2.0 -->

## Setup

> In OVM 2.0 there is no more need for the Remix Optimism plugin, please ignore it. All

Reviewer: I would prefer not even referencing the plugin by name. 
```suggestion
In OVM 2.0, there is no need for any custom Remix plugins. All
```

you need to do is specify [the network information](https://community.optimism.io/docs/infra/networks.html#optimistic-kovan).

To use Remix:

> 1. Log on with your wallet to Optimistic Ethereum (or Optimistic Kovan).

Reviewer: Link to guide on how to do this or link out to chainid.link

[significantly cheaper](https://optimism.io/gas-comparison).

> :warning: We are currently in the process of upgrading to OVM 2.0, but that
> is still work in progress (expected to end 11 NOV 2021). This tutorial is
> already upgraded for OVM 2.0, but parts of it will change during the upgrade
> process.

   <!-- TEMP-OVM2.0 -->

## Setup

In OVM 2.0, there is no need for any custom Remix plugins. All
you need to do is specify [the network information](https://community.optimism.io/docs/infra/networks.html#optimistic-kovan).

To use Remix:

1. Log on with your wallet to Optimistic Kovan (or, eventually,
   Optimistic Ethereum). You can do this using chainid.link:
   - [Optimistic Kovan](https://chainid.link?network=optimism-kovan)
   - [Optimistic Ethereum](https://chainid.link?network=optimism)

1. Browse to [Remix](https://remix.ethereum.org/).
1. Click the run icon (<img src="https://remix-ide.readthedocs.io/en/latest/_images/a-run-icon.png" height="24" valign="top" />).
1. Select the Environment **Injected Web3 Provider**.
1. Accept the connection in the wallet.

> That's it. 

Reviewer 2: Now you can start writing OVM contracts! "That's it!" feels a bit weird here.
